lib/puppet/pops/binder/lookup.rb

```diff
@@ -137,7 +137,14 @@ def self.search_for(scope, type, name, options)
    end
   end
 
-  # This is the method called from the puppet/parser/functions/lookup.rb
```

Comment:
I don't think we want to give the Lookup class this public method since it means it can be used elsewhere to call any function in any scope. Isn't this only used to call `hiera` btw?